docs/ml-clustering.md

```diff
@@ -265,3 +265,38 @@ Refer to the [R API docs](api/R/spark.gaussianMixture.html) for more details.
 </div>
 
 </div>
+
+## Power Iteration Clustering (PIC)
+
+Power Iteration Clustering (PIC) is  a scalable graph clustering algorithm
+developed by <a href=http://www.icml2010.org/papers/387.pdf>Lin and Cohen</a>.
```

Comment:
Hi @huaxingao ,  It seems  the link is not accessible now. This link (http://www.cs.cmu.edu/~frank/papers/icml2010-pic-final.pdf)  seems working.

Reply:
@shahidki31 Changed. Thanks!